package.json

```diff
@@ -0,0 +1,8 @@
+{
+	"private": true,
```

Comment:
What does `occ` do when it sees `private:"true"`?
Does it remove it?

Reply:
`occ` generates a new `package.json` from the `bower.json`, and only merges specific fields from an existing `package.json`. it doesn't even look at `private`. reference: https://github.com/Financial-Times/origami-component-converter/blob/fd7a8813a6d3437f4a8f54c5944f32c88fb51f14/lib/npm.js#L59-L78